app/screens/members_modal/index.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+
+import {observeProfileLongPresTutorial} from '@queries/app/global';
+import {observeCurrentUserId} from '@queries/servers/system';
+import {observeTeammateNameDisplay} from '@queries/servers/user';
+
+import MembersModal from './members_modal';
+
+import type {WithDatabaseArgs} from '@typings/database/database';
+
+const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
+    return {
+        teammateNameDisplay: observeTeammateNameDisplay(database),
+        currentUserId: observeCurrentUserId(database),
```

Comment:
this two props are already present in the parent component, it think it just makes sense to pass them down.

Reply:
done.